code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -150,6 +150,159 @@
 
 //Spooky special loot
 
+// Beserker armor
+
+#define MAX_BERSERK_CHARGE 100
+#define PROJECTILE_HIT_MULTIPLIER 1.5
+#define DAMAGE_TO_CHARGE_SCALE 0.75
+#define CHARGE_DRAINED_PER_SECOND 5
+#define BERSERK_DAMAGE_REDUCTION 0.6
+#define BERSERK_ATTACK_SPEED_MODIFIER 0.5
+#define BERSERK_COLOUR rgb(149, 10, 10)
+
+/obj/item/clothing/suit/hooded/berserker
+	name = "champion's hardsuit"
+	desc = "Voices echo from the hardsuit, driving the user insane. Is not space-proof."
+	icon_state = "hardsuit-berserker"
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/resonator, /obj/item/mining_scanner, /obj/item/t_scanner/adv_mining_scanner, /obj/item/gun/energy/kinetic_accelerator, /obj/item/pickaxe, /obj/item/twohanded/spear)
+	armor = list(MELEE = 30, BULLET = 15, LASER = 10, ENERGY = 10, BOMB = 150, BIO = 0, RAD = 0, FIRE = INFINITY, ACID = INFINITY)
```

Comment:
just checking you adjusted these for the armour refactor

Reply:
Yes, these were adjusted with armour refactor in mind